### ISSUE TITLE
Remove < and > (&lt;and &gt;) in all fields

### ISSUE THREAD

**Issue by xhero** [rism-ch/muscat#349]
Many fields in around the dataset still contain &lt;and &gt, they have to be converted to < and > or will not display correctly (eg. institution 16353, source 456082627, etc)

**Comment by lpugin** [comment]
Yes, we should remove all the HTML markup from the content. I can also see that <br> is used in 245. What we would need to do is split them into one 245 and then repeated 246. OK?

@HirschSt how do you want to proceed? This might be too much for this version. Maybe you can just remove all of them for now?

**Comment by HirschSt** [comment]
We probably have many html entities in various fields in the records, for the moment I can try to erase <br> and &gt and &lt; but in later releases we should find a way to replace them or to display them correctly; @lpugin can you explain why htmlentities are bad with rails (aside from incorrect display)?

**Comment by lpugin** [comment]
Yes, I agree that we should look at this for the next release. 

The problem is mostly having html entities in MARC and this is not directly related to Rails.

**Comment by lpugin** [comment]
PS We should distinct the issue of using < and >, which we could just replace with [ and ] for example, and html entities. So for now the fix could be to just replace < with [ and > with ].

**Comment by HirschSt** [comment]
@lpugin ok

**Comment by HirschSt** [comment]
This should be opened again because we have a difference now between records of the migration pool and new record. Eg. in http://muscat.rism.info/admin/sources/456081660 we have "[[L']] oeuvre" (as discussed previously), in http://muscat.rism.info/admin/sources/1001023286 there is "<<Das>> hölzerne Schwert. Skizzen"; I would propose to change (with a maintenance script) the imported records to the angle bracked version, eg  "<<L'>> oeuvre" to have everything in line.

**Comment by xhero** [comment]
Since the original issue was fixed, and this was never risen again, I think we can safely close this historical ticket!
